### DISCUSSION TITLE
Seata 有没有办法结合编程式事务使用？或者有没有不使用注解的方法？

### DISCUSSION THREAD

**Discussion by nikbobo** [apache/incubator-seata#5827] [General]
看了下，主流事务的使用都是 @GlobalTransactional 和 @Transactional，有没有可能像 Spring 的事物模版一样使用 Seata 而不是用注解？
类似 TransactionalTemplate 这种方法，GlobalTransactionalTemplate？好像没有。

**Comment by funky-eyes** [comment]
https://seata.io/zh-cn/docs/user/api.html
